checks/prgenv/mpi.py

```diff
@@ -53,16 +31,23 @@ class MpiInitTest(rfm.RegressionTest):
     )
     prebuild_cmds += ['module list']
     time_limit = '2m'
-    maintainers = ['JG', 'AJ']
+    maintainers = ['@jgphpc']
     tags = {'production', 'craype'}
 
     @run_after('init')
     def set_cpp_flags(self):
         self.build_system.cppflags = self.cppflags[self.required_thread]
 
+    @run_after('setup')
+    def skip_if_no_mpi(self):
+        self.skip_if(self.current_partition.name.startswith('login'),
+                     'skip login partition')
+        self.skip_if(self.current_environ.name.startswith('builtin'),
+                     'skip builtin pe')
```

Comment:
Same for the environment. I would prefer to add a feature for all the mpi envs.

Reply:
https://github.com/eth-cscs/cscs-reframe-tests/pull/77/commits/6131d0eb876873aa4bfe70de4cd91597642f3f79